### PR TITLE
Users

### PR DESCRIPTION
**Pull request by Tyu-cr**

### CODE REVIEW

File: backend/users/views.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    except User.DoesNotExist:
+        return Response(status=status.HTTP_404_NOT_FOUND)
+
+    if request.method == 'GET':
+        serializer = UserSerializer(user)
+        return Response(serializer.data)
+
+
+@api_view(['POST'])
+def register(request, format=None):
```
Comment (ruca905) [inline backend/users/views.py:32]: I think, that is obvious that only register app in our backend can trigger this endpoint (creation of new user == register new user). So we should somehow forbid this action for all users except register module. But for now this is not a big problem, security is out of MVP, so if you dont know how to do it, you can create an issue with low priority and forget about this problem for some time

File: backend/users/models.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
+from django.db import models
+
+
+# Create your models here.
+
+class User(models.Model):
+    userId = models.IntegerField(null=False, blank=False)
+    passwordMd5hash = models.CharField(max_length=300, null=True, blank=True)
+    name = models.CharField(max_length=100, null=False, blank=False)
+    phoneNumber = models.IntegerField(null=False, blank=False)
```
Comment (ruca905) [inline backend/users/models.py:10]: I think, we should make phone number string typed. Now, i`ll update the openapi branch, pull it to get this change

**Review by ruca905 — CHANGES_REQUESTED** [review]

File: backend/users/serializers.py [hunk 3]
```diff
@@ -0,0 +1,39 @@
+from rest_framework import serializers
+from .models import User
+
+
+class UserSerializer(serializers.Serializer):
+    id = serializers.IntegerField(read_only=True)
+    userId = serializers.IntegerField()
```
[the remark this answers is not in the document]
Comment (Tyu-cr) [reply backend/users/serializers.py:7]: first id - number of row in db
second id - user id

[the remark this answers is not in the document]
Comment (ruca905) [reply backend/users/serializers.py:7]: These are the same entities

**Review by ruca905 — DISMISSED** [review]

**Review by ruca905 — APPROVED** [review]

**Pull request merged**